include/deal.II/lac/block_vector.h

```diff
@@ -517,6 +518,16 @@ namespace internal
   } /* namespace LinearOperator */
 } /* namespace internal */
 
+
+/** mark dealii::BlockVector< Number > as serial vector
+ *
+ * @author Uwe Köcher, 2017
```

Comment:
good question, I think we should not do it because it will probably break on windows. ;-)